packages/minisign/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://jedisct1.github.io/minisign/
+TERMUX_PKG_DESCRIPTION="A dead simple tool to sign files and verify signatures"
+TERMUX_PKG_LICENSE="ISC"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.9
+TERMUX_PKG_DEPENDS="libsodium"
+TERMUX_PKG_BUILD_DEPENDS="pkg-config"
```

Comment:
There should be no need to specify pkg-config as build depends (the pkg-config that is packaged for termux is not used by host when cross-compiling)